packages/cover-traffic-daemon/package.json

```diff
@@ -23,7 +22,7 @@
   },
   "files": [
     "lib",
-    "default-environment.json"
```

Comment:
why did you remove this file?

Reply:
there is no `default-environment.json` in that directory, so we don't need to include it. Maybe this is an error?

Reply:
Correct. The functionality of keeping the info in a separate file has been changed. Instead its now part of the releases configuration.